frontend/src/services/productService.ts

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import {parseError} from './errorService';
+import {IProduct} from "../interfaces/IProduct";
+
+
+export const getAllProducts = (token: string) => {
+    return axios({
+        method: 'get',
+        url: `/api/product`,
+        headers: {"Content-Type": "application/json", "Authorization": `Bearer ${token}`}
```

Comment:
you could swap the source code into a variable and use it for all of your api methods
```suggestion
        method: 'get',
        url: `/api/product`,
        headers: {"Content-Type": "application/json", "Authorization": `Bearer ${token}`}
```

Reply:
absolutely right